app/View/CoPeople/fields.inc

```diff
@@ -1246,17 +1249,18 @@
                     </td>
                     <td>
                       <?php
-                        print $this->Html->link(_txt($e && !$es ? 'op.edit' : 'op.view'),
+                        print $this->Html->link(_txt($e && !$es && $org_e ? 'op.edit' : 'op.view'),
```

Comment:
Why dont you use $permissions['edit'] instead of $org_e?

Reply:
Done